azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -109,6 +109,34 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							ValidateFunc: validation.IntBetween(1, 100),
 						},
 
+						"max_count": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+
+						"min_count": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+
+						"type": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
```

Comment:
Needs validation but i think master has this property now

Reply:
YEs master had it now.